src/arfs_entities.ts

```diff
@@ -245,6 +256,8 @@ export class ArFSPublicFolder extends ArFSFileOrFolderEntity {
 			txId,
 			unixTime,
 			0,
+			stubTransactionID,
+			'application/json',
```

Comment:
Wouldn't hate a constant for this. Can totally see us misspelling it somewhere at some point.